packages/utils/src/network/decomposeDataUri.ts

```diff
@@ -51,7 +51,7 @@ export interface DecomposedDataUri {
  * @param {string} dataUri - the data URI to check
  * @return {PIXI.utils.DecomposedDataUri|undefined} The decomposed data uri or undefined
```

Comment:
What about the doc for DecomposedDataUri? Should that be updated too?

Reply:
Ah yes i didnt see that!